data/graphics.js

```diff
@@ -13592,6 +13618,34 @@ var BattleMoveAnims = {
 			var attacker = args[0];
 			var defender;
 
+			battle.bgElem.animate({
+				position: 'absolute',
+				left: -5,
+				right: 5,
+			}, 75).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: -5,
+				right: 5
+			}, 100).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: -5,
+				right: 5
+			}, 100).animate({
+				left: 5,
+				right: -5
+			}, 100).animate({
+				left: 0,
+				right: 0,
+			}, 100, function () {
+				battle.bgElem.animate({
+					position: 'static'
+				});
```

Comment:
Isn't it supposed to be `absolute` to begin with?

I'm actually 80% sure the background has a lot of padding for a reason.